src/index.js

```diff
@@ -1,31 +1,40 @@
 import * as babel from '@babel/core';
 import { createFilter } from 'rollup-pluginutils';
-import createPreflightCheck from './preflightCheck.js';
-import helperPlugin from './helperPlugin.js';
-import { addBabelPlugin, escapeRegExpCharacters, warnOnce } from './utils.js';
-import { RUNTIME, EXTERNAL, HELPERS } from './constants.js';
+import preflightCheck from './preflightCheck.js';
+import bundledHelpersPlugin from './bundledHelpersPlugin.js';
+import { addBabelPlugin, escapeRegExpCharacters } from './utils.js';
+import { RUNTIME, EXTERNAL, BUNDLED, INLINE, HELPERS } from './constants.js';
 
 const unpackOptions = ({
 	extensions = babel.DEFAULT_EXTENSIONS,
+	skipBabelHelpersCheck = false,
```

Comment:
we need to:
1. either rename this, because it disables more than a helper check
2. introduce a second flag (or more) for other disabled things

Reply:
What else does it disable? It seems to only disable the `preflightCheck`, right?

Reply:
Yes, that's the only things disabled - but preflighCheck is not only about the helpers check. It also performs a module check - https://github.com/rollup/rollup-plugin-babel/blob/f644b9a89beeecfe370567d26fd0b45b24f25913/src/preflightCheck.js#L47

Reply:
Ah ok - I understand the distinction now. We could just name it `skipPreflightCheck` and if someone posts an issue with a specific use-case for a more granular option we can add that in a minor release.

Reply:
Yeah, `skipPreflightCheck` sounds good.